### PR TITLE
aB3kT6-genesis-validator-data

### PR DESCRIPTION
**Pull request by alchimestre**

### CODE REVIEW

**Comment by raj-shekhar1** [comment]
Can you please add a static IP instead of domain, Auotnity does not supports DNS yet, though in next releases there will be an option to update the static IP.

```
"enode://bce186f2b956818d96b80293b8a78233c77daa84004e33304ed55ece1c6650b7a3a21249b005d362ff8c5fb919d7c8ac707e075b24379d3252be558fc16a83f2@validator-0.autonity.mainnet.kiln.fi",
```

**Review by raj-shekhar1 — APPROVED** [review]
Looks good :+1: 
```
./ethkey verifypop 0x52a7c9895545c6e6b221af7a2ac708f0eefa09fd enode://bce186f2b956818d96b80293b8a78233c77daa84004e33304ed55ece1c6650b7a3a21249b005d362ff8c5fb919d7c8ac707e075b24379d3252be558fc16a83f2@51.159.206.200:62222 0x0aB543903ff68F1CdF948D3bDEf0706C14eAaB9B 0xa5e5189151fd06c91c11943cd1c861a0bf801f4feffe290284edec8d848d6078b2f216e6098efb1c4b4d615c735facc8 0xe9233127c2cb9c0693998c5d37e41f53a3851b4e24e01505351f9f5ab64c328b73fa259d27ef237b9c1075037a0e188755b1c382a5f53639bdc4ea87258d3cfe005b7cdb0ba95f76247dd80c74a5d88dc864b3a7ada09283794a3967dc92807b43658a7d76cc89857307cc3996ab0ddcde32674ec38a28a4ef69937472808eeb77008572c8c7d9378ba5c5b32208245db03c854e12aee7c2dfc63e47bbaf6fb7ffae2c4b5226ee887fcc6ac7e0e8fef9cbae16f862e69a84b90524c885d2bbd104987b1235bc6c3793e3358228dc5b3cf17dd0642b98fc16ab2501df592126879077
===== POP Validation Results =====
Node Key:       true
Oracle Key:     true
Consensus Key:  true
```

File: genesis-validator-data/template.json
Comment (spenot09) [inline genesis-validator-data/template.json:1]: @Vloupmon all looks good can you just avoid deleting this please?

**Review by spenot09 — APPROVED** [review]

**Comment by Polkaponika** [comment]
>

**Review by Polkaponika — APPROVED** [review]

**Pull request merged**